### DISCUSSION TITLE
Chatlog disappear after using a lazer tourney client

### DISCUSSION THREAD

**Discussion by hayasaka-san-kr** [ppy/osu#19736] [Q&A]
### Type

Other

### Bug description

There was a same problem before 2 weeks ago, but I had same issue today.

I broadcasted tourney with using lazer client two weeks ago, and my all of chatlog disappeared.

And I had same issue right after using a tourney client today.

Can you fix this problem and restore my chatlist?

https://osu.ppy.sh/users/5247450

### Screenshots or videos

![Screenshot_20220813-231919_Chrome](https://user-images.githubusercontent.com/77833547/184498750-ced42ac6-9e77-42ce-8e97-9de16b70bc7a.jpg)

### Version

osu lazer 7/21

### Logs

No logs

**Comment by peppy** [comment]
What page are you looking at there? It doesn't look like the chat listing page.

**Reply by hayasaka-san-kr** [reply]
![Screenshot_20220814-230854_Chrome](https://user-images.githubusercontent.com/77833547/184540882-1c77fab4-86db-4151-a7a5-43aa2f9062cd.jpg)

**Reply by peppy** [reply]
Please provide the *url* and provide logs from the tournament client next time this happens. I have no idea what is going on here.;
